### PR TITLE
Confirmation modal

### PR DESCRIPTION
**Pull request by nikolabojkovic**

### CODE REVIEW

File: src/app/modals/confirmation-modal/confirmation-modal.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <button type="button" class="close pull-right" aria-label="Close" (click)="closeModal()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <h3>Confirm you request</h3>    
+  </div>
+  <div class="modal-body">
+      <div class="confirmation-message">{{message}}</div>
+  </div>
+  <div class="modal-footer">
+      <button class="btn btn-default" (click)="cancel()">Cancel</button>
```
Comment (RajicA) [inline src/app/modals/confirmation-modal/confirmation-modal.component.html]: Lets pass to modal these labels also.

File: src/app/modals/confirmation-modal/confirmation-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+@Component({
+    selector: 'confirmation-modal',
+    templateUrl: './confirmation-modal.component.html',
+    styleUrls: [ './confirmation-modal.component.css' ]
+})
+export class ConfirmationModalComponent implements OnInit {
+
+    message: string;
+    @Output() onResponse = new EventEmitter<boolean>();
+    registrationValidationErrors: any = {};
```
Comment (RajicA) [inline src/app/modals/confirmation-modal/confirmation-modal.component.ts]: This should be removed.

File: src/app/modals/modals.modul.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```
Comment (RajicA) [inline src/app/modals/modals.modul.ts]: Name of this file should be modals.module.ts

File: src/app/training-programs/training-programs.component.ts [hunk 4]
```diff
@@ -36,6 +38,10 @@ export class TrainingProgramsComponent implements OnInit {
     next() {
 
         // change this code below - redirect to next step and save information
-        alert('next step');
+        this.confirmationModal.showModal("Go to the next step?");
```
Comment (RajicA) [inline src/app/training-programs/training-programs.component.ts]: Use LITERALS const

File: src/app/modals/confirmation-modal/confirmation-modal.component.html (hunk 1 above)
Comment (nikolabojkovic) [reply src/app/modals/confirmation-modal/confirmation-modal.component.html]: Which labels?

**Pull request merged**